aiida/post-init.sh

```diff
@@ -5,11 +5,12 @@
 # Parse first/last name from GIT_AUTHOR_NAME
 # See https://stackoverflow.com/a/17841619/1069467
 function join_by { local d=$1; shift; local f=$1; shift; printf %s "$f" "${@/#/$d}"; }
-user=($GIT_AUTHOR_NAME)
+user="$(git config user.name)"
 first_name=${user[0]}
-last_name=`join_by ' ' "${user[@]:1}"`
+last_name=$(join_by ' ' "${user[@]:1}")
+email="$(git config user.email)"
 
-project_dir=`pwd`
+project_dir=$(pwd)
 
 cat > aiida_config.yaml <<EOF
 store_path: "${project_dir}/repo"
```

Comment:
I realize this is not what you are addressing in this PR, so it's fine if you take it elsewhere, but `repo/` should be added to `.gitignore` (and maybe called something more descriptive?)